tests/fv3_conf/cpld_control_run.IN

```diff
@@ -55,19 +55,36 @@ if [ $WARM_START = .F. ]; then
 # Restart
 else
   # Restart files
-  cp ../${DEP_RUN}${RT_SUFFIX}/RESTART/coupler.res ./INPUT
-  cp ../${DEP_RUN}${RT_SUFFIX}/RESTART/fv_*.nc ./INPUT
-  cp ../${DEP_RUN}${RT_SUFFIX}/RESTART/sfc_data*.nc ./INPUT
-  cp ../${DEP_RUN}${RT_SUFFIX}/RESTART/phy_data*.nc ./INPUT
-  cp ../${DEP_RUN}${RT_SUFFIX}/RESTART/MOM*.nc ./INPUT
+  cp -r ../${DEP_RUN}${RT_SUFFIX}/RESTART/${RESTART_FILE_PREFIX}.* ./INPUT
```

Comment:
The same here. We might have extra ${RESTART_FILE_PREFIX}.* files in ./INPUT directory

Reply:
Yes, I think that is right. I forgot to delete this line and also in the cpld_bmark_run.IN file.

Reply:
Now that I look at it, I'm not quite sure. I tried to follow what is done in gfdlmp_run.IN. But I'm using a restart_interval of 6 hours so in a 1 day run, I get multiple timestamped FV3 restart files. I'm copying in here only the one which corresponds to the one I want to use in the restart test.